cmd/queue/main.go

```diff
@@ -224,18 +265,18 @@ func buildProbe(logger *zap.SugaredLogger, encodedProbe string, autodetectHTTP2
 }
 
 func buildServer(ctx context.Context, env config, probeContainer func() bool, stats *network.RequestStats, logger *zap.SugaredLogger,
-	ce *queue.ConcurrencyEndpoint) (server *http.Server, drain func()) {
+	ce *queue.ConcurrencyEndpoint, enableTLS bool) (server *http.Server, drain func()) {
```

Comment:
If TLS is enabled, will we be executing probes twice and tracking two different sets of container health?

This seems like something to solve for alpha, so maybe leave a TODO somewhere in here about that?